can/bus.py

```diff
@@ -24,6 +25,15 @@ class BusState(Enum):
     ERROR = auto()
 
 
+class ErrorState(Enum):
+    """The fault confinement state in which CAN interface can be."""
+
+    BUS_OFF = auto()
+    ERROR_PASSIVE = auto()
```

Comment:
Newcomers should be able to find some documentation on when these error states occur.

Reply:
I added a more detailed description to the property BusABC.error_statistics as i think the ErrorState class docstring will not be visible in the documentation.
 Please correct me if i'm wrong.